core/src/modules/ops/destination.ts

```diff
@@ -40,53 +44,79 @@ export namespace DestinationOps {
   /**
    * Export all the Group Members of the Groups that this Destination is Tracking
    */
-  export async function exportGroupMembers(
-    destination: Destination,
-    force = false
-  ) {
-    const group = await destination.$get("group");
-    if (group) return group.run(force, destination.id);
+  export async function exportMembers(destination: Destination, force = false) {
+    if (destination.collection === "group") {
+      const group = await destination.$get("group");
+      if (group) return group.run(force, destination.id);
+    } else if (destination.collection === "model") {
+      const model = await destination.$get("model");
+      if (model) return model.run(force, destination.id);
+    } else {
+      throw new Error(`cannot export members for a ${destination.collection}`);
```

Comment:
Notably this now throws an error if we're not tracking anything, let's make sure that we can handle this from wherever this is called. For example, when deleting the destination.